myJS.js

```diff
@@ -2,33 +2,70 @@ document.addEventListener("DOMContentLoaded", function () {
     const apiKey = '00b5bc3025c06669c4aecf0a745e058b';
     const searchForm = document.getElementById("search-form");
     const searchInput = document.getElementById("search-input");
-    let searchResults = document.getElementById("search-results");
+    const searchResults = document.getElementById("search-results");
+    let currPage=1;
+    const perPage=10;
 
     searchForm.addEventListener("submit", function (e) {
```

Comment:
for 'click' event you used arrow function. I think it's a good choice. 
it's also important to stay consistent